daemon/stores/peas.py

```diff
@@ -9,18 +9,21 @@ class PeaStore(ContainerStore, AiohttpMixin):
 
     _kind = 'pea'
 
-    async def add_in_partial(self, uri: str, params: Dict, **kwargs) -> Dict:
+    async def add_in_partial(
+        self, uri: str, params: Dict, envs: Optional[Dict] = {}, **kwargs
+    ) -> Dict:
         """Sends `POST` request to `partial-daemon` to create a Pea/Pod.
 
         :param uri: uri of partial-daemon
         :param params: json payload to be sent
+        :param envs: environment variables to be passed into partial pea
         :param kwargs: keyword args
         :return: response from mini-jinad
         """
         return await self.POST(
             url=f'{uri}/{self._kind}',
             params=None,
-            json=params,
+            json={'pea': params, 'envs': envs},
```

Comment:
```suggestion
            json={self._kind: params, 'envs': envs},
```
`add_in_partial` in `PodStore` is not required.